core/templates/search/search_pages_results.html

```diff
@@ -109,15 +109,6 @@ <h4 class="term">{{ paginator.count }} results
 <form class="form-inline search_results_form" action="" method="GET">
   <div class="row searhc_results_option_set">
     <div class="col-md-3">
-      <label for="state">
```

Comment:
My goal was just to hide it, not eliminate it.  Partially to allow others to put in states if need be (though I can't imagine that'll be a common use-case), partially because there's value to having the data stored just so we have it if we for some reason need it in the future, and partially because it would probably be a *lot* of work to actually rip it out fully.